Call for Volunteers 2025 (#100)
djangocon/2024.djangocon.us: src/_content/posts/call-for-2025-volunteers.md

---
author: DjangoCon US Communications Team
category: General
date: 204-10-18 06:00:00
layout: post
title: "Call for Volunteers: DjangoCon US 2025"
---

> As our program chair, Drew Winstel, mentioned in his opening remarks today, we would love to have more volunteers for DjangoCon US 2025. Expect to hear details on dates and venues within the next month or so.

Reviewer: If @PeterGrand is happy to plug this during the Wednesday opening remarks, put him in here instead of me. Also maybe be a little more vague about when we'll announce the conference

Reviewer 2: I'm going to remove this paragraph entirely. Peter's got a lot on his plate and I'd rather let him fit things in as needed. The callout doesn't do much in my opinion.

We have some [documentation about the various roles](https://djangocon.github.io/djangocon-us-docs/), but here's a quick overview of the teams at DjangoCon US.

The DEFNA board will not leave you stranded; we have a lot of documentation to help you on your way, including rough checklists of tasks needed for each team.

Ideally, all committees will have a chair and co-chair to allow for one or the other to take vacations, handle day job duties, and so forth. This list is by no means comprehensive, but it should hopefully give you an idea of what each committee is responsible for.

## Conference Leadership

> The conference chair and co-chair lead the conference, including coordinating between teams and making executive decisions (including approaching the DEFNA board when applicable). Expect to be in meetings with the venue and online conference provider, weekly organizer check-ins, monthly reports to the DEFNA board, and lots of questions on Slack. You'll also lead the volunteer kickoff meetings in early 2023 (ideally mid-to-late January) and recruit organizers as you see fit. You'll need to recommend an online conference provider to the DEFNA board (might be part of a sponsorship deal).

Reviewer: Early 2025

At the conference, you'll be making speeches at the start of each day and the end-of-conference wrap-up in addition to coordinating setup, teardown, and everything in between.

These two roles have the highest time commitment and the most visibility. If you want to help people get things done and get up on stage, these are the roles for you.

## Program Committee

Size: 2-4

> The program committee is responsible for making sure that the schedule of tutorials and talks is the best it can be.

Reviewer: Note that this probably has the second highest time commitment outside the conference chair

### Pre-conference responsibilities

> - Start the call for proposals process (ideally opening in February or March 2025)

Reviewer: ```suggestion
- Start the call for proposals process (ideally opening in February 2025)
```

- Coordinate with the Opportunity Grants team to schedule the opening and closing of the CFP
- Publish blog posts announcing the CFP, speaker mentors, and speaker resources
- Recruit keynote speakers (the DEFNA board will offer suggestions, but if you have a message you'd like the keynotes to send, you have lots of latitude here, but keep in mind conference diversity goals)

> - Recruit speaker mentors (start with the 2019, 2021, and 2022 reviewers and bring in more as you like)

Reviewer: ```suggestion
- Recruit speaker mentors (start with the 2022-24 reviewers and bring in more as you like)
```

- Review proposals as they come in for anonymization (ideally, recruit one member of the program committee who will not be reviewing talks to do the anonymization before opening talks for review)

> - Recruit talk reviewers (the DEFNA board will send you the 2022 reviewers, but more reviewers are encouraged, especially from diverse backgrounds). There will likely be over 100 submissions, so having more reviewers lessens the load for individual talks

Reviewer: ```suggestion
- Recruit talk reviewers (the DEFNA board will send you the 2024 reviewers, but more reviewers are encouraged, especially from diverse backgrounds). There will likely be over 100 submissions, so having more reviewers lessens the load for individual talks
```

### At-conference responsibiities

- Answer questions from grant recipients and the DEFNA board

## Community Team

Size: 2-4

### Pre-conference responsibilities

- Schedule orientation
- Arrange childcare and lactation room
- Coordinate visa applications
- Improve accessibility of the venue and conference as a whole

### At-confrence responsibilities

- Post sign-ups for lactation room and distribute keys as needed
- Collect receipts for childcare reimbursement

## Sponsorship Committtee

Size: 2-4

The sponsorship committee recruits sponsors which help offset the cost of running DCUS.

### Pre-conference responsibilities

- Create and announce sponsorship prospectus (we have templates, and DEFNA will help with specifics)
- Recruit sponsors
- Coordinate contract signing with DEFNA
- Consider awarding community sponsorships (non-financial) to other non-profit Python conferences and related communities (local Python meetups near the conference will love the exposure)
- Coordinate shipping of sponsor swag for the swag bag
- Lay out sponsor tables (on a venue map)
- Work with sponsor chair/co-chair to handle sponsor shout-outs if needed

> - Order banners to go behind the speakers on stage

Reviewer: Paging @sodevious to make sure this is up to date

### At-conference responsibilities

- Direct sponsors to their booths/tables and swag
- Ensure that booths are reserved for sponsors who have reserved a booth, and are not open for others
- Field complaints/questions from sponsors

## Code of Conduct Committee

> Size: 4-6

Reviewer: Paging @VeldaKiara to review this section

### Pre-conference responsibilities

- Suggest modifications/improvements to the CoC
- Assist program committee with pre-conference review of recorded online talks

### At-conference responsibilities

- Field reports and investigate them, including recommending actions to the DEFNA board
- Follow-up on reports, including conversations with both reporters and those who have been reported, and facilitate the appropriate response
- Write a CoC transparency report to recap reports (heavily anonymized) and actions taken

## Social Events Committee

Size: 2-3

The social events committee is responsible for scheduling the "after hours" events at DCUS to give people chances to socialize and unwind after a long day of talks.

### Pre-conference responsibilities

- Find a venue for the speaker appreciation dinner and coordinate contract signing with DEFNA
- Plan opening reception (if sponsored, this will become a bigger event compared to previous years)
- Plan any other fun events like board game night

### At-conferece responsibilities

- Work with venue on any work needed for social events

> ## Venue Committee

Reviewer: @PeterGrand input here would be great

Size: 1-2

The venue committee is responsible for coordinating plans between the other organizing teams and the venue(s).

### Pre-conference responsibilities

- Work with event venue to select and lay out talk, tutorial, and sprint rooms
- Lay out the extra rooms needed for the conference (lactation room, quiet room, organizers room, speaker green room, lunch/break areas)
- Work with event venue and DEFNA to select catering menus (and providers if we're not contractually obligated to use one)
- Work with hotel to coordinate room blocks
- Work with venue to plan logistics (sponsor gear shipping, signage, etc.)
- Allocate reward rooms in coordination with DEFNA board
- Order supplies for restroom stocking

### At-conference responsibilities

- Review room guest list against ticket list to claim reward nights
- Handle any last-minute requests from vendors/sponsors/organizers/etc.
- Notify venue staff of any needs (bathroom cleanliness, meals to accommodate special diets, etc.)

## Audio/Video (A/V) Committee

Size: 2-3

> The A/V committee is responsible for making everything look good. They handle coordination between the on-site crew, the online streaming platform, and the post-conference processing teams and upload the results to YouTube after the conference.

Reviewer: @adamfast I know you have this on lockdown, but I want to make sure this is reasonably current

### Pre-conference responsibilities

- Select on-site video production crew
- Select post-conference processing crew
- Coordinate both of the above teams with the online platform
- Work with the venue team to ensure high-speed wired internet access to the on-site video crew
- Transfer pre-recorded talks to the post-conference processing crew and then the online conference provider
- Procure a wide variety of dongles for presenters to hook their laptops up to the projectors
- Procure projectors and screens if the venue doesn't already have them, including one for the green room so speakers can test their setup

### At-conference responsibilities

- Help on-site crew set up equipment and ensure streaming works
- Get dongles to the right rooms as needed
- Help keynote speakers get presentation mode working if needed
- Answer lots and lots of questions

> ## Swag Committee

Reviewer: More from @sodevious here

### Pre-conference responsibilities

- Design and order attendee t-shirt (with sponsor logos)
- Design and order swag bags (with sponsor logos)
- Design and order speaker gifts
- Design and order DCUS-branded swag (stickers/magnets/etc.)
- Order masks and COVID tests (if needed)
- Order lanyards, badges, and badge flags

### At-conference responsibilities

- Recruit volunteers to stuff swag bags on Sunday afternoon
- Get T-shirts and swag bags to registration desk
- Get speaker gifts to the program team for distribution

## Volunteer Committee

Size: 2

The volunteer committee is responsible for cultivating new volunteers, which can become organizers in future years.

### Pre-conference responsibilities

- Determine what roles need on-site volunteer support (session chairs, session managers, registration desk, etc.)
- Set up sign-up for on-site volunteers and online volunteers (moderators/cheerleaders)
- Draft announcements for calls for volunteers

### At-conference responsibilities

- Tell volunteers where to go
- Give volunteers tools to succeed

## Communications Committee

Size: 2-4

> The communications committee is responsible for getting the word out about DCUS.

Reviewer: @tim-schilling you and Jason have this section for sure

Reviewer 2: Yeah, I looked it over. It seems pretty accurate.

> Have you heard the expression "never tweet" (usually said on Twitter)? This is the opposite of that. Always tweet.

Reviewer: ```suggestion
Have you heard the expression "never tweet" (usually said on the site formerly known as Twitter)? This is the opposite of that. Always tweet.
```

Reviewer 2: I removed this sentence. I think this phrase has passed on now.

Reviewer: Good by me

### Pre-conference responsibilities

- Publish announcements and reminders on social media, the conference blog, and more (Django News, Python Community News, etc.)
- Send copy to the Django Software Foundation to get major news events on the Django blog
- Review, edit, and suggest copy for the other teams' announcements
- Tweet. Lots of tweeting.
- Select an "official emoji" for the conference (typically suggested by the design package for the year's website)
- Boost and retweet speakers and organizers talking about the conference

> - Schedule tweets announcing talks and events

Reviewer: To-do: rework this to be less focused on Twitter

Reviewer 2: I reworked it to say tweeting and tooting

Reviewer: Teehee

### At-conference responsibilities

- Pay attention to the `#djangocon` topic on social media and mentions to `@djangocon` on Twitter
- Boost speakers and organizers
- Answer questions as they come up
- Run the `#help-desk` channel on the conference attendee slack

## Website Committee

Size: 2-3

The website committee is responsible for updating the website as the conference transitions through phases (CFP coming soon, CFP open, CFP closed, schedule released, hotel block closed, conference over, etc.), reviewing pull requests, and adding content (e.g. new sponsors). This requires the most coding of any team aside from maybe the program team.

### Pre-conference responsibilities

- Update static pages for CFP status
- Update static pages for venue information
- Review pull requests from other teams

> - Add sponsors as directed by sponsorship team

Reviewer: @sodevious this one is also all yours

## Automation Team

This team is more free-flowing and writes scripts to make teams' lives easier. Responsibilities vary based on the year, an if you have ideas for special projects for the future, have at it!

## Sprints Team

Size: 2-4

### Pre-conference responsibilies

- Write blog posts announcing sprints
- Invite people to launch special sessions (e.g. Carlton's ["Getting Started Contributing to Django"](https://2022.djangocon.us/sprints/getting-started-contributing-to-django/))
- Ensure we have signage and power equipment

### At-conference responsibilities

- Coordinate help desk
- Direct people to lunches and breaks

## Wrap-up

> To sign up for any of the teams (or just say "put me to work"), send an email to hello@djangocon.us and we'll put you on the list for the 2023 kickoff meeting. We would love to have you! You do not have to have previous organizing experience (although we might prefer it for some of the leadership roles).

Reviewer: ```suggestion
To sign up for any of the teams (or just say "put me to work"), send an email to hello@djangocon.us and we'll put you on the list for the 2025 kickoff meeting. We would love to have you! You do not have to have previous organizing experience (although we might prefer it for some of the leadership roles).
```

Author: Done

---
author: DjangoCon US Communications Team
category: General

> date: 204-10-18 06:00:00

Reviewer 2: ```suggestion
published_datetime: 2025-08-23 06:00:00-04:00
```

### Pre-conference responsibilities

- Start call for grants process (concurrent with CFP)
- Publish blog posts announcing grants process
- Request grants from Django Software Foundation and Python Software Foundation (the DEFNA board will help you fill out the forms)
- Recruit grant reviewers (other organizers and DEFNA members are happy to help)
- Review submissions
- Coordinate with program team about grants to speakers
- Notify grant recipients in waves (concurrent with program notifications)
- Communicate with recipients on how to get reimbursed (we'll have templates)
- Give DEFNA a lit of recipients and amounts
- Approve reimbursements as requests come in
- Coordinate with visa (community) team as needed to approve invitation letter requests
- Write transparency report giving a high-level overview of how the grants were awarded

### At-conference responsibilities

- Answer questions from grant recipients and the DEFNA board

## Community Team

Size: 2-4

### Pre-conference responsibilities

- Schedule orientation
- Arrange childcare and lactation room
- Coordinate visa applications

> - Improve accessibility of the venue and conference as a whole

Reviewer 3: When I was gathering info on my responsibilities, improving accessibility didn't seem to be a high priority. Do we want to leave that on the list of responsibilities? (To be clear, I think it would be great to do and I'm sure there's room for improvement.)

Potential responsibilities to add:
- Organize hackathon
- Organize community interviews

Reviewer 2: Personally, I think the hackathon actually belongs under the Social team and the community interviews probably under the Communications team. That said, how it worked out this year was fantastic. It was a good example of a person seeing something that was of interest of them and pushing it forward.

Size: 2-3

The social events committee is responsible for scheduling the "after hours" events at DCUS to give people chances to socialize and unwind after a long day of talks.

### Pre-conference responsibilities

- Find a venue for the speaker appreciation dinner and coordinate contract signing with DEFNA
- Plan opening reception (if sponsored, this will become a bigger event compared to previous years)
- Plan any other fun events like board game night

### At-conferece responsibilities

- Work with venue on any work needed for social events

## Venue Committee

Size: 1-2

The venue committee is responsible for coordinating plans between the other organizing teams and the venue(s).

### Pre-conference responsibilities

- Work with event venue to select and lay out talk, tutorial, and sprint rooms
- Lay out the extra rooms needed for the conference (lactation room, quiet room, organizers room, speaker green room, lunch/break areas)
- Work with event venue and DEFNA to select catering menus (and providers if we're not contractually obligated to use one)
- Work with hotel to coordinate room blocks
- Work with venue to plan logistics (sponsor gear shipping, signage, etc.)

> - Allocate reward rooms in coordination with DEFNA board

Reviewer 4: ```suggestion
- Allocate rooms for keynotes and AV personnel in coordination with DEFNA board
- Communicate needs, deadlines and policies to other teams
```

- Order supplies for restroom stocking

### At-conference responsibilities

> - Review room guest list against ticket list to claim reward nights
> - Handle any last-minute requests from vendors/sponsors/organizers/etc.
> - Notify venue staff of any needs (bathroom cleanliness, meals to accommodate special diets, etc.)

Reviewer 4: ```suggestion
- Handle closeout of the room block with hotel and DEFNA board
- Handle any last-minute requests from vendors/sponsors/organizers/etc.
- Coordinate with venue staff about issues or needs (bathroom cleanliness, meals to accommodate special diets, etc.)
```

## Audio/Video (A/V) Committee

Size: 2-3

The A/V committee is responsible for making everything look good. They handle coordination between the on-site crew, the online streaming platform, and the post-conference processing teams and upload the results to YouTube after the conference.

### Pre-conference responsibilities

- Select on-site video production crew
- Select post-conference processing crew
- Coordinate both of the above teams with the online platform
- Work with the venue team to ensure high-speed wired internet access to the on-site video crew
- Transfer pre-recorded talks to the post-conference processing crew and then the online conference provider
- Procure a wide variety of dongles for presenters to hook their laptops up to the projectors
- Procure projectors and screens if the venue doesn't already have them, including one for the green room so speakers can test their setup

### At-conference responsibilities

- Help on-site crew set up equipment and ensure streaming works
- Get dongles to the right rooms as needed
- Help keynote speakers get presentation mode working if needed
- Answer lots and lots of questions

> ## Online Team

Reviewer 2: @blulady I attempted to define the online responsibilities as best I could. It could be improved though for sure

- Coordinate with online moderators
- Engage with audience as able to
- Help facilitate any other online activities

## Swag Committee

Size: 1-2

The swag committee is responsible for everything speakers and attendees take home from the conference (aside from knowledge!).

### Pre-conference responsibilities

- Design and order attendee t-shirt (with sponsor logos)
- Design and order swag bags (with sponsor logos)
- Design and order speaker gifts
- Design and order DCUS-branded swag (stickers/magnets/etc.)
- Order masks and COVID tests (if needed)
- Order lanyards, badges, and badge flags

### At-conference responsibilities

- Recruit volunteers to stuff swag bags on Sunday afternoon
- Get T-shirts and swag bags to registration desk
- Get speaker gifts to the program team for distribution

## Volunteer Committee

Size: 2

The volunteer committee is responsible for cultivating new volunteers, which can become organizers in future years.

### Pre-conference responsibilities

- Determine what roles need on-site volunteer support (session chairs, session managers, registration desk, etc.)
- Set up sign-up for on-site volunteers and online volunteers (moderators/cheerleaders)

> - Draft announcements for calls for volunteers

Reviewer 5: - Draft announcements for calls for volunteers **and handle communication with volunteers**

### At-conference responsibilities

- Tell volunteers where to go

> - Give volunteers tools to succeed

Reviewer 5: can we replace this with something like: 
- Lead volunteer orientation on the first morning of the conference
- Create and provide volunteer instructions for each applicable area
- Tell volunteers where to go

### Pre-conference responsibilities

- Update static pages for CFP status
- Update static pages for venue information
- Review pull requests from other teams
- Add sponsors as directed by sponsorship team

## Automation Team

This team is more free-flowing and writes scripts to make teams' lives easier. Responsibilities vary based on the year, an if you have ideas for special projects for the future, have at it!

## Sprints Team

Size: 2-4

### Pre-conference responsibilities

- Write blog posts announcing sprints
- Invite people to launch special sessions (e.g. Carlton's ["Getting Started Contributing to Django"](https://2022.djangocon.us/sprints/getting-started-contributing-to-django/))
- Ensure we have signage and power equipment

### At-conference responsibilities

- Coordinate help desk
- Direct people to lunches and breaks

## Wrap-up

> To sign up for any of the teams (or just say "put me to work"), email hello@djangocon.us, and we'll put you on the list for the 2025 kickoff meeting. We would love to have you! You do not have to have previous organizing experience (although we might prefer it for some leadership roles).

Reviewer 6: Are we removing the "Health & Safety" committee next year?

Reviewer: I didn't think to include it in my 2023 version and it didn't get added here. Want to put something together for it? (and/or @jcjudkins)

Reviewer 6: ```
## Health & Safety Team

Size: 1-2

The health and safety committee enacts the health and safety policy each year. The policy itself is decided by DEFNA and is not part of this committee's responsibilities. 

### Pre-conference responsibilities

- Order COVID tests and masks (if needed per policy)
- Ensure that the health & safety check-in has volunteers
- Coordinate the health and safety blog posts with the Comms team

### At-conference responsibilities

- Spend some volunteer time at the health & safety check-in (if it is part of the policy that year)
- Answer health & safety questions as needed

```

---
author: DjangoCon US Communications Team
category: General
date: 204-10-18 06:00:00
layout: post
title: "Call for Volunteers: DjangoCon US 2025"
---

> As our program chair, Drew Winstel, mentioned in his opening remarks today, we would love to have more volunteers for DjangoCon US 2025. Expect to hear details on dates and venues within the next month or so.

Reviewer 2: ```suggestion
A great way to give back to the Django community is to help organize the next one. Perhaps you want to see something at DjangoCon US 2025. If that's you, we'd love to support your efforts in making that a reality. Or maybe you'd like to make sure the next people have the same wonderful experience you had this year. Regardless of your reason, we would love to have more volunteers for DjangoCon US 2025.
```

## Conference Leadership

The conference chair and co-chair lead the conference, including coordinating between teams and making executive decisions (including approaching the DEFNA board when applicable). Expect to be in meetings with the venue and online conference provider, weekly organizer check-ins, monthly reports to the DEFNA board, and lots of questions on Slack. You'll also lead the volunteer kickoff meetings in early 2025 (ideally mid-to-late January) and recruit organizers as you see fit. You'll need to recommend an online conference provider to the DEFNA board (might be part of a sponsorship deal). This year we used [Venueless](https://venueless.org/), though that's not a requirement.

At the conference, you'll be making speeches at the start of each day and the end-of-conference wrap-up in addition to coordinating setup, teardown, and everything in between.

These two roles have the highest time commitment and the most visibility. If you want to help people get things done and get up on stage, these are the roles for you.

## Program Committee

Size: 2-4

> The program committee is responsible for making sure that the schedule of tutorials and talks is the best it can be.

Reviewer 2: ```suggestion
The program committee is responsible for making sure that the schedule of tutorials and talks is the best it can be. There is both a high degree of influence over the tutorials and talks and a high time commitment when you're a chair of the program committee.
```

Reviewer 2: @drewbrew do you want this called out explicitly? I tried to make it not sound like an awful gig.

Reviewer 2: @drewbrew do you want to change this before we post it later this week?

Reviewer: sorry, I missed this comment. I like it

- Coordinate with online moderators
- Engage with audience as able to
- Help facilitate any other online activities

## Swag Committee

Size: 1-2

The swag committee is responsible for everything speakers and attendees take home from the conference (aside from knowledge!).

### Pre-conference responsibilities

- Design and order attendee t-shirt (with sponsor logos)
- Design and order swag bags (with sponsor logos)
- Design and order speaker gifts
- Design and order DCUS-branded swag (stickers/magnets/etc.)
- Order masks and COVID tests (if needed)
- Order lanyards, badges, and badge flags

### At-conference responsibilities

- Recruit volunteers to stuff swag bags on Sunday afternoon
- Get T-shirts and swag bags to registration desk
- Get speaker gifts to the program team for distribution

## Volunteer Committee

Size: 2

The volunteer committee is responsible for cultivating new volunteers, which can become organizers in future years.

### Pre-conference responsibilities

- Determine what roles need on-site volunteer support (session chairs, session managers, registration desk, etc.)
- Set up sign-up for on-site volunteers and online volunteers (moderators/cheerleaders)

> - Draft announcements for calls for volunteers

Reviewer 2: ```suggestion
- Draft announcements for calls for volunteers
- Communicate and coordinate with volunteers
```

### At-conference responsibilities

> - Tell volunteers where to go
> - Give volunteers tools to succeed

Reviewer 2: ```suggestion
- Lead volunteer orientation on the first morning of the conference
- Create and provide volunteer instructions for each applicable area
- Tell volunteers where to go
```

The website committee is responsible for updating the website as the conference transitions through phases (CFP coming soon, CFP open, CFP closed, schedule released, hotel block closed, conference over, etc.), reviewing pull requests, and adding content (e.g. new sponsors). This requires the most coding of any team aside from maybe the program team.

### Pre-conference responsibilities

- Update static pages for CFP status
- Update static pages for venue information
- Review pull requests from other teams
- Add sponsors as directed by sponsorship team

## Automation Team

This team is more free-flowing and writes scripts to make teams' lives easier. Responsibilities vary based on the year, an if you have ideas for special projects for the future, have at it!

## Sprints Team

Size: 2-4

### Pre-conference responsibilities

- Write blog posts announcing sprints
- Invite people to launch special sessions (e.g. Carlton's ["Getting Started Contributing to Django"](https://2022.djangocon.us/sprints/getting-started-contributing-to-django/))
- Ensure we have signage and power equipment

### At-conference responsibilities

- Coordinate help desk
- Direct people to lunches and breaks

> 

Reviewer 2: ```suggestion

## Health & Safety Team

Size: 1-2

The health and safety committee enacts the health and safety policy each year. The policy itself is decided by DEFNA and is not part of this committee's responsibilities.

### Pre-conference responsibilities
- Order COVID tests and masks (if needed per policy)
- Ensure that the health & safety check-in has volunteers
- Coordinate the health and safety blog posts with the Comms team

### At-conference responsibilities
- Spend some volunteer time at the health & safety check-in (if it is part of the policy that year)
- Answer health & safety questions as needed

```

---
author: DjangoCon US Communications Team
category: General

> published_datetime: 2025-08-23 06:00:00-04:00

Reviewer 2: ```suggestion
published_datetime: 2024-09-25 13:00:00-04:00
```
